assets/js/app.js

```diff
@@ -182,7 +182,16 @@ const listeners = {
     delete uploads[e.detail.uuid]
   },
   "phx:upload:pause": (e) => { uploads[e.detail.uuid].pause() },
-  "phx:upload:resume": (e) => { uploads[e.detail.uuid].resume() }
+  "phx:upload:resume": (e) => { uploads[e.detail.uuid].resume() },
+  "phx:gallery:preload": (e) => {
+    const img = new Image();
+    img.onload = function() {
+      const preloadedList = document.getElementById(e.detail.to);
+      preloadedList.appendChild(img);
```

Comment:
It would be good to clear it after 30 sec, so looking through 100 images wont make them kept in this list